devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDeviceVariableTreeNames.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BIPEDAL_LOCOMOTION_fRAMEWORK_YARP_ROBOT_LOGGER_DEVICE_VTN_H
+#include <iostream>
```

Comment:
I guess it should be ``string``

```suggestion
#include <string>
```

Reply:
Fixed in [bf7fe13](https://github.com/ami-iit/bipedal-locomotion-framework/commit/bf7fe13ca037a598119aaab783e0961903dd546f)